### ISSUE TITLE
ui5-link does not honor the a aria-controls and aria-expanded 

### ISSUE THREAD

**Issue by hkshetry** [SAP/ui5-webcomponents#3546]
### **Bug Description**
ui5-link does not honor the a aria-controls and aria-expanded. html hyper link does not add aria-controls and aria-expanded  attribute so ui5-link  can't use as collapse and expend link 

### **Steps to Reproduce**
1. go to https://codesandbox.io/s/ui5-webcomponents-forked-l2hcb?file=/index.html
2.  Turn on screen reader and Navigate to link. screen reader does not announce "Expand Less link"

### **Isolated Example**
https://codesandbox.io/s/ui5-webcomponents-forked-l2hcb?file=/index.html

Expected behavior
aria-controls = "textId" and aria-expanded = true should be present in HTML DOM Element

### **Context**
- UI5 Web Components version: 0.31.12
- OS/Platform: NA
- Browser: NA
- Affected component: ui5-link

### **Priority**
- [ ] Low
- [ ] Medium
- [x] High
- [ ] Very High

### **Stakeholder Info** (if applicable)

- Organization: SuccessFactor

**Comment by fifoosid** [comment]
@ilhan007 take a look at this feature request

**Comment by rchrdchn** [comment]
Hi @DMihaylova @petyabegovska - do we have a proposed fix and date the fix will be in the main branch? If so, what's estimated date? Thanks.

**Comment by petyabegovska** [comment]
Hi @rchrdchn,

This is a new feature request for the ui5-link component and the use case is under discussion by our Accessibility experts. I will update you as soon as we have more information on a date for the main branch.

Best regards,
Petya

**Comment by ilhan007** [comment]
Hello @hkshetry the Link now provides the accessibilityAttributes property to set `aria-expand`, `aria-haspopup`
with this change https://github.com/SAP/ui5-webcomponents/pull/4591 available with 1.1.0

**Comment by ilhan007** [comment]
I will let Peta, Boyan to decide if this issue can be closed.

**Comment by petyabegovska** [comment]
The issue can be closed.
